api/src/main/java/org/pmiops/workbench/db/model/DbInstitution.java

```diff
@@ -78,21 +94,69 @@ public void setOrganizationTypeOtherText(String organizationTypeOtherText) {
     this.organizationTypeOtherText = organizationTypeOtherText;
   }
 
+  /** Builder method to help streamline the building of a DbInstitution. */
+  public DbInstitution organizationTypeOtherText(String organizationTypeOtherText) {
+    this.organizationTypeOtherText = organizationTypeOtherText;
+    return this;
+  }
+
+  @OneToMany(mappedBy = "institution", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
EAGER fetching is key to making this work correctly.

Reply:
Huh, you don't have to give it the key column names. Neat.